google/cloud/spanner_dbapi/connection.py

```diff
@@ -119,6 +181,14 @@ def run_sql_in_snapshot(self, sql, params=None, param_types=None):
             return list(res)
 
     def get_table_column_schema(self, table_name):
+        """Get table column schema.
+
+        :type table_name: str
+        :param table_name: Name of the table.
+
+        :rtype: dict
+        :returns: Column description.
```

Comment:
Table columns description, I guess, as you're returing a `dict` with description for every column